.ci/compute_projects.py

```diff
@@ -130,12 +132,11 @@ def _add_dependencies(projects: Set[str]) -> Set[str]:
 def _compute_projects_to_test(modified_projects: Set[str], platform: str) -> Set[str]:
     projects_to_test = set()
     for modified_project in modified_projects:
-        # Skip all projects where we cannot run tests.
-        if modified_project not in PROJECT_CHECK_TARGETS:
-            continue
         if modified_project in RUNTIMES:
             continue
-        projects_to_test.add(modified_project)
+        # Skip all projects where we cannot run tests.
```

Comment:
Why is this getting moved around if it doesn't cause test differences in `compute_projects_tests.py`?

Reply:
Without this change, we won't even consider `.ci` because it doesn't have an entry in PROJECT_DEPENDENCIES.

Reply:
I added a test for '.ci'.